client/src/components/HistoryViewer/HistoryViewerVersionDetail.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import FormBuilderLoader from 'containers/FormBuilderLoader/FormBuilderLoader';
+import HistoryViewerVersionList from './HistoryViewerVersionList';
+import Loading from './Loading';
+import { versionType } from 'types/versionType';
+
+class HistoryViewerVersionDetail extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleLoadingSuccess = this.handleLoadingSuccess.bind(this);
+
+    this.state = {
+      loading: true,
+    };
+  }
+
+  /**
+   * When the form builder has finished loading the schema, change the state
+   * to remove the loading indicator
+   */
+  handleLoadingSuccess() {
+    this.setState({
+      loading: false
+    });
+  }
+
+  render() {
+    const { handleSetCurrentVersion, schemaUrl, version } = this.props;
+    const { loading } = this.state;
+
+    return (
+      <div className="history-viewer">
+        <HistoryViewerVersionList
+          extraClass="history-viewer__table--current"
+          versions={[version]}
+          handleSetCurrentVersion={handleSetCurrentVersion}
```

Comment:
I don't know if we need to be passing this down the tree through several components. The advantage of redux is that you can inject this at the point in the tree where the concern is. I'd prefer to see it at the `HistoryListVersion` level. Just add a container export to that file.